getting-started-guides/csp/gcp/spark-gpu/README.md

```diff
@@ -30,34 +30,38 @@ all prerequisites installed:
     `<CLUSTER_NAME>`. Before the init script fully merged into `<dataproc-initialization-actions>` bucket, user need to copy the sparkgpu initialization script into a accessible GCS and into following structure. Ubuntu is recommended as CUDA support ubuntu, debian could be used by modifying `image-version` and `linux-dist` accordingly.
 
     ```
-    /$STORAGE_BUCKET/sparkgpu/rapids.sh
-    /$STORAGE_BUCKET/sparkgpu/internal/install-gpu-driver*.sh
-    ```  
-
+    /$GCS_BUCKET/sparkgpu/rapids.sh
+    /$GCS_BUCKET/sparkgpu/internal/install-gpu-driver*.sh
+    ```
+    
     ```bash
-    export CLUSTER_NAME=sparkgpu
+    export CLUSTER_NAME=my-gpu-cluster
     export ZONE=us-central1-b
     export REGION=us-central1
-    export STORAGE_BUCKET=dataproc-initialization-actions
+    export GCS_BUCKET=my-bucket
+    export INIT_ACTIONS_BUCKET=my-bucket
     export NUM_GPUS=2
     export NUM_WORKERS=2
-    
+     
     gcloud beta dataproc clusters create $CLUSTER_NAME  \
         --zone $ZONE \
         --region $REGION \
         --master-machine-type n1-standard-32 \
+        --master-boot-disk-size 50 \
         --worker-accelerator type=nvidia-tesla-t4,count=$NUM_GPUS \
         --worker-machine-type n1-standard-32 \
+        --worker-boot-disk-size 50 \
         --num-worker-local-ssds 1 \
         --num-workers $NUM_WORKERS \
         --image-version 1.4-ubuntu18 \
-        --bucket $STORAGE_BUCKET \
-        --metadata zeppelin-port=8081,INIT_ACTIONS_REPO="gs://$STORAGE_BUCKET",linux-dist="ubuntu" \
-        --initialization-actions gs://$STORAGE_BUCKET/spark-gpu/rapids.sh \
-        --optional-components=ZEPPELIN \
+        --bucket $GCS_BUCKET \
+        --metadata JUPYTER_PORT=8123,INIT_ACTIONS_REPO="gs://$INIT_ACTIONS_BUCKET",linux-dist="ubuntu",
+        GCS_BUCKET="gs://$GCS_BUCKET" \
+        --initialization-actions gs://$INIT_ACTIONS_BUCKET/spark-gpu/rapids.sh \
+        --optional-components=ANACONDA,JUPYTER \
         --subnet=default \
-        --properties 'spark:spark.dynamicAllocation.enabled=false,spark:spark.shuffle.service.enabled=false' \
-        --enable-component-gateway       
+        --properties '^#^spark:spark.dynamicAllocation.enabled=false#spark:spark.shuffle.service.enabled=false#spark:spark.submit.pyFiles=/usr/lib/spark/python/lib/xgboost4j-spark_2.11-1.0.0-Beta2.jar#spark:spark.jars=/usr/lib/spark/jars/xgboost4j-spark_2.11-1.0.0-Beta2.jar,/usr/lib/spark/jars/xgboost4j_2.11-1.0.0-Beta2.jar,/usr/lib/spark/jars/cudf-0.9.1-cuda10.jar' \
```

Comment:
Beta3 and cudf-0.9.1-cuda10.jar should be cudf-0.9.2-cuda10.jar

Reply:
done, also need to change maven url and from `xgboost4j_2.11` to `xgboost4j_2.x`